src/Calculator/Views/UnitConverter.xaml.cpp

```diff
@@ -152,18 +152,18 @@ void UnitConverter::OnOfflineNetworkAccess()
 
 void UnitConverter::SetNormalCurrencyStatus()
 {
-    CurrencySecondaryStatus->Text = L"";
+    VisualStateManager::GoToState(this, NormalCurrencyStatus->Name, false);
 }
 
 void UnitConverter::SetChargesMayApplyStatus()
 {
-    VisualStateManager::GoToState(this, L"ChargesMayApplyCurrencyStatus", false);
+    VisualStateManager::GoToState(this, ChargesMayApplyCurrencyStatus->Name, false);
```

Comment:
nitpicky, but probably a better pattern to set the text before changing the visual state, ie, do the right work before displaying.

Reply:
Thanks for cleaning up my mess 👍 😄

Reply:
Agreed.